hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -0,0 +1,115 @@
+# Pi4 Bootloader Configuration
+
+You can display the current configuration using 
+```
+vcgencmd bootloader_config
+```
+
+To change these bootloader configuration items, you need to extract the configuration segment, make changes, re-insert it, then reprogram the EEPROM with the new bootloader. 
+
+```
+# Extract the configuration file
+cp /lib/firmware/raspberrypi/bootloader/beta/pieeprom-2019-09-10.bin pieeprom.bin
+rpi-eeprom-config pieeprom.bin > bootconf.txt
+
+# Edit the configuration using a text editor e.g. nano bootconf.txt
+
+# If you have a UART cable then setting BOOT_UART=1 will help debug boot issues
+BOOT_UART=1
```

Comment:
Is this bit relevant in this section?

Reply:
Just there as an example of a change you might make, perhaps E.G. is necessary.